internal/cmd/local/local/locate.go

```diff
@@ -0,0 +1,67 @@
+package local
+
+import (
+	"fmt"
+
+	"github.com/pterm/pterm"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/getter"
+	"helm.sh/helm/v3/pkg/repo"
+)
+
+func locateLatestAirbyteChart(chartName, chartVersion string) string {
+	pterm.Debug.Printf("getting helm chart %q with version %q\n", chartName, chartVersion)
+
+	// Helm will consider a local directory path named "airbyte/airbyte" to be a chart repo,
+	// but it might not be, which causes errors like "Chart.yaml file is missing".
+	// This trips up plenty of people, see: https://github.com/helm/helm/issues/7862
+	//
+	// Here we avoid that problem by figuring out the full URL of the airbyte chart,
+	// which forces Helm to resolve the chart over HTTP and ignore local directories.
+	// If the locator fails, fall back to the original helm behavior.
+	if chartName == airbyteChartName && chartVersion == "" {
+		if url, err := getLatestAirbyteChartUrlFromRepoIndex(airbyteRepoName, airbyteRepoURL); err == nil {
+			pterm.Debug.Printf("determined latest airbyte chart url: %s\n", url)
+			return url
+		} else {
+			pterm.Debug.Printf("error determining latest airbyte chart, falling back to default behavior: %s\n", err)
+		}
+	}
+
+	return chartName
+}
+
+func getLatestAirbyteChartUrlFromRepoIndex(repoName, repoUrl string) (string, error) {
+	chartRepo, err := repo.NewChartRepository(&repo.Entry{
+		Name: repoName,
+		URL:  repoUrl,
+	}, getter.All(cli.New()))
+	if err != nil {
+		return "", fmt.Errorf("unable to access repo index: %w", err)
+	}
+
+	idxPath, err := chartRepo.DownloadIndexFile()
+	if err != nil {
+		return "", fmt.Errorf("unable to access repo index: %w", err)
+	}
+
+	idx, err := repo.LoadIndexFile(idxPath)
+	if err != nil {
+		return "", fmt.Errorf("unable to access repo index: %w", err)
```

Comment:
```suggestion
		return "", fmt.Errorf("unable to load index file (%s): %w", idxPath, err)
```

Reply:
Thanks. Committed in 33ff5bc899391056bac6d2211a93f1e015f4aa9d